utils/oscap.8

```diff
@@ -75,6 +75,11 @@ You may specify OVAL Definition files as the last parameter, XCCDF evaluation wi
 Select a particular profile from XCCDF document.
 .RE
 .TP
+\fB\-\-rule RULE\fR
+.RS
+Select a particular rule from XCCDF document (also requires profile to be specified). Only this rule will be evaluated.
+.RE
+.TP
```

Comment:
@shawndwells Now that you mentioned it, I should probably remove the part which says that profile is required to be specified (based on the discussion above). Thanks.